canvas_modules/harness/cypress/integration/canvas/node-insert-in-link.js

```diff
@@ -71,6 +71,30 @@ describe("Test dropping a node on a link with enableInsertNodeDroppedOnLink opti
 			"Binding (entry) node", "outPort", "Execution node", "inPort", 0);
 	});
 
+	it("Tests for drag node over link", function() {
```

Comment:
Please change the description here to be: "Drag a node from the palette onto a link AND the canvas onto a link"